mypy/sharedparse.py

```diff
@@ -6,36 +6,52 @@
 
 _NON_BINARY_MAGIC_METHODS = {
     "__abs__",
+    "__bool__"
     "__call__",
+    "__ceil__",
     "__complex__",
-    "__contains__",
     "__del__",
     "__delattr__",
+    "__delete__",
     "__delitem__",
+    "__dir__",
     "__enter__",
     "__exit__",
     "__float__",
+    "__floor__",
+    # "__get__",  # TODO: breaks testAccessingGenericDescriptorFromClassBadOverload
     "__getattr__",
     "__getattribute__",
     "__getitem__",
+    "__hash__",
     "__hex__",
+    "__index__",
     "__init__",
     "__init_subclass__",
+    "__instancecheck__",
     "__int__",
     "__invert__",
     "__iter__",
     "__len__",
+    "__length_hint__",
     "__long__",
+    "__missing__",
     "__neg__",
     "__new__",
+    "__not__",
```

Comment:
As far as I know, this method is not special. It's included in the `operator` module though but some of the names are specific to the module and aren't really special.